### PR TITLE
Add TYPO3 Flow unit tests to framework parity check

### PR DESCRIPTION
**Pull request by tmotyl**
Add class override for Flow framework
Use phpUnit shipped with the framework (v 4.5)
Use branch 3.0 of Flow, which contains HHVM fixes already

Resolves: https://github.com/facebook/hhvm/issues/3007

### CODE REVIEW

**Comment by fredemmott** [comment]
Reverting, does not work: Fatal error: Uncaught exception 'InvalidArgumentException' with message 'Parameter must be an array or an instance of Traversable' in :
Stack trace:
#0 /data/fio/hhvm/hphp/test/frameworks/Framework.php(810): HH\Set->addAll()
#1 /data/fio/hhvm/hphp/test/frameworks/Framework.php(131): Framework->findTests()
#2 /data/fio/hhvm/hphp/test/frameworks/framework_class_overrides/Flow.p

File: hphp/test/frameworks/frameworks.yaml [hunk 1]
```diff
@@ -139,6 +139,13 @@
     commit: 7dc9a43cb856debc22fe158a1144018e16f7a637
     install_root: fastroute
     test_root: fastroute
+  flow:
+    url: https://git.typo3.org/Flow/Distributions/Base.git
+    branch: '3.0'
+    commit: 300f50f02bf41c25677111bd0c8962643b16bff8
+    install_root: flow
+    test_root: flow
```
Comment (fredemmott) [inline hphp/test/frameworks/frameworks.yaml]: This is incorrect. You need flow/Build/BuildEssentials/PhpUnit

This is why it broke.

Paths in the phpunit xml are interpretted relative to the test root. The paths in yours are relative to the directory containing the phpunit xml.

**Comment by kitsunet** [comment]
I would still like to see Flow in the framework list. It delivers a phpunit configuration (to use a specific bootstrap), seems that was not used. What can I do to get this setup?

**Comment by tmotyl** [comment]
I have updated the branch with changes suggested here. 
Can you reopen the pull request, or should I create a new one?
https://github.com/tmotyl/hiphop-php/commit/da2daad861341b76d6a5dafcdf41b79260650fb7

**Comment by fredemmott** [comment]
A new pull request would be best to make this easier to track :)

**Comment by tmotyl** [comment]
done https://github.com/facebook/hhvm/pull/5104

**Pull request closed without merging**